google/api_core/exceptions.py

```diff
@@ -116,15 +119,16 @@ class GoogleAPICallError(GoogleAPIError, metaclass=_GoogleAPICallErrorMeta):
     This may be ``None`` if the exception does not match up to a gRPC error.
     """
 
-    def __init__(self, message, errors=(), response=None):
+    def __init__(self, message, errors=(), details=(), response=None):
         super(GoogleAPICallError, self).__init__(message)
         self.message = message
         """str: The exception message."""
         self._errors = errors
+        self._details = details
         self._response = response
 
     def __str__(self):
-        return "{} {}".format(self.code, self.message)
+        return "{} {} {}".format(self.code, self.message, self.error_details)
```

Comment:
Having the empty list show up is distracting and not useful.  Better would be:
```suggestion
        details = self.error_details
        if details:
            return "{} {} {}".format(self.code, self.message, details)
        else:
            return "{} {}".format(self.code, self.message)
```